common/djangoapps/util/usermautic.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from django.conf import settings
+
+from mautic import MauticBasicAuthClient, Contacts
+
+logger = logging.getLogger(__name__)
+
+
+COUNTRIES = {
```

Comment:
why dont you importthe countries from django countries, its already used in our app stack.

Reply:
Yeah, but I'm using this because Mautic doesn't accept a country's code, and it only allows entries on country full names that they already defined. So what I'm doing is getting the country code from the django-countries field and then looking it up in Mautic-countries to submit the country name to the API.